doc/src/vpr/fasm.rst

```diff
@@ -0,0 +1,264 @@
+FPGA Assembly (FASM) Output Support
+===================================
+
```

Comment:
I like the documentation, but should probably go under something like: `doc/src/utils/fasm`